incubator/patroni/templates/NOTES.txt

```diff
@@ -0,0 +1,29 @@
+Getting Started:
+
+Patroni can be accessed via port 5432 on the following DNS name from within your cluster:
+{{ template "fullname" . }}.{{ .Release.Namespace }}.svc.cluster.local
+
+To get your password for superuser or admin run:
+
+    # admin user password
+    printf $(printf '\%o' `kubectl get secret --namespace {{ .Release.Namespace }} {{ template "fullname" . }} -o jsonpath="{.data.password-admin[*]}"`);echo
+
+    # superuser password
+    printf $(printf '\%o' `kubectl get secret --namespace {{ .Release.Namespace }} {{ template "fullname" . }} -o jsonpath="{.data.password-superuser[*]}"`);echo
+
+To connect to your database:
+
+1. Run an Ubuntu pod that you can use as a client:
+
+    kubectl run -i --tty ubuntu --image=ubuntu:16.04 --restart=Never -- bash -il
```

Comment:
Yeah that works too, my main concern was to avoid having to install psql.

If we want to make the steps really easy, it looks like you can provide the `PGPASSWORD` env var (https://www.postgresql.org/docs/current/static/libpq-envars.html). At the risk of making it look very complicated and ugly, we could have something like:
```
PGPASSWORD=$(printf $(printf '\%o' `kubectl get secret --namespace {{ .Release.Namespace }} {{ template "fullname" . }} -o jsonpath="{.data.password-admin[*]}"`);echo)
kubectl run -i --tty --rm psql --image=postgres --env "PGPASSWORD=$PGPASSWORD" --command -- psql -U superuser -h {{ template "fullname" . }}.{{ .Release.Namespace }}.svc.cluster.local postgres
```